src/solidity/types/StringType.js

```diff
@@ -40,7 +40,7 @@ function format (decoded) {
   var value = decoded.value
   value = value.replace('0x', '').replace(/(..)/g, '%$1')
   var ret = {
-    // length: decoded.length, // unneeded, only dynamicBytes uses length
+    length: decoded.length,
```

Comment:
I can't remember why i was convinced that the length here is never used, but it doesn't matter.